.github/workflows/python-release-tests.yml

```diff
@@ -0,0 +1,87 @@
+name: Test Anemoi Packages for Release
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  quality:
+    uses: ecmwf/reusable-workflows/.github/workflows/qa-precommit-run.yml@v2
+    with:
+      skip-hooks: "no-commit-to-branch"
+
+  checks:
+    if: ${{ github.event_name == 'pull_request' && github.event.pull_request.user.login == 'DeployDuck'}}
```

Comment:
Putting this check down here means that the workflow and quality job will run on every PR. I would add a path filter at the top so that the whole workflow only runs on release PRs, like so:

```
on:
  pull_request:
    types: [opened, synchronize, reopened]
    paths:
      - ".release-please-manifest.json"
```